src/gui/advancedsettings.cpp

```diff
@@ -499,6 +539,15 @@ void AdvancedSettings::loadAdvancedSettings()
     comboBoxSeedChokingAlgorithm.addItems({tr("Round-robin"), tr("Fastest upload"), tr("Anti-leech")});
     comboBoxSeedChokingAlgorithm.setCurrentIndex(static_cast<int>(session->seedChokingAlgorithm()));
     addRow(SEED_CHOKING_ALGORITHM, tr("Upload choking algorithm"), &comboBoxSeedChokingAlgorithm);
+    item(SEED_CHOKING_ALGORITHM, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("Controls the seeding unchoke behavior. The available options are:<br><br>"
+                    "<strong>Round-robin</strong>: which round-robins the peers that are unchoked when seeding. "
```

Comment:
you should not place html tags into `tr()`

Reply:
Is there a way to format it without splitting it in many strings?

Reply:
Actually I saw qt designer generate this: `<html><head/><body><p>text</p></body></html>`, so this is probably fine.